platformio.ini

```diff
@@ -9,6 +9,7 @@ framework = arduino
 board_build.filesystem = littlefs
 lib_deps =
 	mathieucarbou/ESPAsyncWebServer@^3.6.0
+    knolleary/PubSubClient@^2.8
```

Comment:
This module appears to be abandoned and has lots of [issues](https://github.com/knolleary/pubsubclient/issues?q=sort%3Aupdated-desc+is%3Aissue+is%3Aopen)

Is there a better alternative we can use that is still being maintained?